gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -697,6 +693,13 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
       gchar **options = NULL;
       guint i, num_options;
 
+      if (!g_regex_match_simple (REGEX_STAND_OPTION, filter->option, 0, 0)) {
```

Comment:
How about setting `G_REGEX_CASELESS` as compile_options to ignore the case?

Reply:
updated